.github/workflows/dependencies/dependencies_nvhpc21-7.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Copyright 2021 Axel Huebl
+#
+# License: BSD-3-Clause-LBNL
+
+set -eu -o pipefail
+
+sudo apt-get -qqq update
+sudo apt-get install -y \
+    build-essential     \
+    ca-certificates     \
+    cmake               \
+    environment-modules \
+    gnupg               \
+    pkg-config          \
+    wget
+
+wget -q https://developer.download.nvidia.com/hpc-sdk/21.7/nvhpc-21-7_21.7_amd64.deb \
+        https://developer.download.nvidia.com/hpc-sdk/21.7/nvhpc-2021_21.7_amd64.deb
+sudo apt-get update
+sudo apt-get install -y ./nvhpc-21-7_21.7_amd64.deb ./nvhpc-2021_21.7_amd64.deb
```

Comment:
```
After this operation, 8,983 MB of additional disk space will be used.
```
:eyes:

Reply:
If you wanna go the container route, NVIDIA also provides [this](https://ngc.nvidia.com/catalog/containers/nvidia:nvhpc)

Reply:
Thanks! Will use this for the reproducer for Nvidia.
Unfortunately, that container is of the same size or even a bit larger than hand-picking the primary packages we need here.

Reply:
There's Shifter on Cori to deal with Docker containers... https://docs.nersc.gov/development/shifter/how-to-use/